client/src/components/tags/TagSelector/useTagSelector.ts

```diff
@@ -47,7 +48,7 @@ export default function useTagSelector({ maximum, tagsById }: UseTagSelector = {
 
 	// TODO: If tags are passed through props (=p.tagsById), they take priority over all the
 	// user's tags (=t.tags.tagsById), We need to rename the variables to make that clear.
-	const tags = Object.values(tagsById ?? tagsData?.byId ?? []);
+	const tags = byIdAsList(tagsById ?? tagsData?.byId ?? new Map());
```

Comment:
Don't need the "new Map()" because I changed the byIdAsList to handle undefined input.